src/main/java/edu/csula/datascience/acquisition/ReadJsonTest.java

```diff
@@ -0,0 +1,185 @@
+	    
+		int year=2001;
+		
+		for (int i=year;i<=2015;i++ ){
+		String jsonString = callURL("https://data.cityofchicago.org/resource/6zsd-86xi.json?$limit=100000&year="+i,i);
+	
+
+		System.out.println("============== success for =========== " + i + "============================");
+		}
+		//String jsonString = callURL("https://data.cityofchicago.org/resource/6zsd-86xi.json?$limit=75000&year=2015",2015);
+
+		//https://data.cityofnewyork.us/api/views/nc67-uf89/rows.json?accessType=DOWNLOAD
+		//System.out.println(jsonString);
+		
+	}
+	
+@SuppressWarnings("unchecked")
+public static String callURL(String myURL,int year) {
+	System.out.println("Requested URL:" + myURL);
+	StringBuilder sb = new StringBuilder();
+	URLConnection urlConn = null;
+	InputStreamReader in = null;
+	try {
+		
+		
+		URL url = new URL(myURL);
+		urlConn = url.openConnection();
+		if (urlConn != null)
+			urlConn.setReadTimeout(60 * 1000);
+		if (urlConn != null && urlConn.getInputStream() != null) {
+			in = new InputStreamReader(urlConn.getInputStream(),
+					Charset.defaultCharset());
+			BufferedReader bufferedReader = new BufferedReader(in);
+			if (bufferedReader != null) {
+				int cp;
+				while ((cp = bufferedReader.read()) != -1) {
+					sb.append((char) cp);
+
+
+				}
+				bufferedReader.close();
+			}
+		}
+		
+		System.out.println("b4 dtabase");
+		
+		Mongo mongo = new Mongo("localhost", 27017);
+		DB db = mongo.getDB("Crimedata");
+		
+		DBCollection collection = db.getCollection(Integer.toString(year));
+
+		List<DBObject> a = new ArrayList<DBObject>();
+		
+		a = (List<DBObject>) JSON
+				.parse(sb.toString());
+		
+		BulkWriteOperation builder = collection.initializeUnorderedBulkOperation();
+        for(DBObject doc : a)
+        {
+            builder.insert(doc);
+        }
+        com.mongodb.BulkWriteResult result = builder.execute();
+        
+		
+		/*DBObject dbObject = (DBObject) JSON
+				.parse(sb.toString());*/
+
+		/*for (int i=0; i < dbObject.toString().length(); i++) {
+			collection.insert(dbObject(i));
+		}*/
+		
+		System.out.println("data inserted");
+
+		DBCursor cursorDoc = collection.find();
+		while (cursorDoc.hasNext()) {
+			System.out.println(cursorDoc.next());
+		}
+
+		System.out.println("Done");	
+	    in.close();
+	} catch (Exception e) {
+
+		System.out.println("2");
+		throw new RuntimeException("Exception while calling URL:"+ myURL, e);
+	}
+
+
+	return sb.toString();
+}
+
+	public static String readFile(String filename) {
+	    String result = "";
+	    try {
+	        BufferedReader br = new BufferedReader(new FileReader(filename));
+	        StringBuilder sb = new StringBuilder();
+	        String line = br.readLine();
+	        while (line != null) {
+	            sb.append(line);
+	            line = br.readLine();
+
+	            DBObject dbo = (DBObject) com.mongodb.util.JSON.parse('{'+line+'}');
+	            List<DBObject> list = new ArrayList<>();
+	            list.add(dbo);

```

Comment:
I don't understand why you parse a file, add this item to list which contain single item and insert as a list contain single item.